### DISCUSSION TITLE
Cooldown for all players in custom alias

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#4410] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
Hi, what about adding way to set cooldown for all players into custom aliases? So the command could be executed after specific time, but cooldown will be activated if it's used by anyone.
### Basic explanation as to why this feature is needed and/or useful to have: ###
I wanted to make my moderator team to give reward from events, but to stop abusing, I wanted to make cooldown for ALL of them to just 1 day. So limit using this command for everyone for 1 whole day.

**Comment by CloudeLecaw** [comment]
A nice idea, maybe it also can have a Feedback meassage? Like "this command was used allready by PLAYER, please wait x secounds"
Also if would be nice not just to have it for custom alais, able to have it for specialized commands, so it may can be used in ics too :)

**Comment by bobhenl** [comment]
And couldn't it be made another way somehow for now?
